modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/toolbar/contributor/util/DLPortletToolbarContributorUtil.java

```diff
@@ -22,17 +22,14 @@
 
 import javax.portlet.PortletRequest;
 
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
-
 /**
  * @author Roberto Díaz
  */
-@Component(service = DLPortletToolbarContributorHelper.class)
-public class DLPortletToolbarContributorHelper {
+public class DLPortletToolbarContributorUtil {
```

Comment:
Hey @MrJohn1911, transform it in a base class and make its usages extend it. Thx